Tests/FunctionalTests/StaticBinaryLibrary.swift

```diff
@@ -0,0 +1,34 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+import DriverSupport
+import PackageModel
+import TSCBasic
+import Testing
+import _InternalTestSupport
+
+@Suite
+struct StaticBinaryLibraryTests {
+    @Test
+    func staticLibrary() async throws {
+        try await fixture(name: "BinaryLibraries") { fixturePath in
+            let (stdout, stderr) = try await executeSwiftRun(
```

Comment:
issue: Thanks for writing this test using SwiftTesting, however, there is currently an issue in `_InternalTestHelpers` that will cause the `executeSwift*(...)` functions to fail.  We should convert this to XCTestCase until the issue is resolved.